web-ng/src/app/components/form-field-editor/form-field-editor.component.ts

```diff
@@ -233,6 +233,6 @@ export class FormFieldEditorComponent implements OnInit, OnChanges {
   }
 
   get labelControl() {
-    return this.formFieldGroup.get('label');
+    return this.formFieldGroup.get('label')!;
```

Comment:
! might not be required here as it is always going to return control as it is present part of formFieldGroup

Reply:
Unfortunately formFieldGroup.get() is defined as being able to can return undefined, so we need to check this someplace or we get compile errors.